### DISCUSSION TITLE
Where to find EBS America on livestreaming sites?

### DISCUSSION THREAD

**Discussion by AVGCFan1995** [iptv-org/community#961] [Q&A]
Hey mods, can you find any livestream links for EBS America because the TV channel is very hard-to-find outside of DirecTV most likely due to the lack of popularity or interest making the shows, bumpers, and promos are rare to find nowadays. If unknown some EBS shows were broadcast in either Korean with or without English subtitles, English dubs, English voice-overs, or title translations.

Here's the links.

https://www.lyngsat.com/tvchannels/kr/EBS-America.html

https://www.wikidata.org/wiki/Q122332313

https://streamingtvguides.com/Channel/EBSAMER

https://www.ontvtonight.com/guide/listings/channel/302353609/ebs-america

https://www.tvpassport.com/series/polong-polong-pololo/5687103

https://www.tvpassport.com/series/semi-and-magic-cube/3694989

https://www.youtube.com/watch?v=75QxmLKPxZ8

https://global.ebs.co.kr/global/channels/satelliteIpTv

https://blog.naver.com/ebsstory/50187022377

https://artofchannelbranding.com/2016/04/29/ebs-channel/

Here's the logo.

![Channels_IPTV_05](https://github.com/iptv-org/iptv-org.github.io/assets/143818591/40925663-1442-4d5d-ae29-f392f3ee5bfa)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
